03/app.js

```diff
@@ -1,58 +1,68 @@
 const books = {
-    '978-83-7278-000-3': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Philosopher\'s Stone',
-            pl: 'Harry Potter i Kamień Filozoficzny',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
-    },
-    '83-7278-007-2': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Chamber of Secrets',
-            pl: 'Harry Potter i Komnata Tajemnic ',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
-    },
+	'978-83-7278-000-3': {
+		author: 'J.K. Rowling',
+		title: {
+			en: "Harry Potter and the Philosopher's Stone",
+			pl: 'Harry Potter i Kamień Filozoficzny',
+		},
+		translator: {
+			en: null,
+			pl: 'Andrzej Polkowski',
+		},
+	},
+	'83-7278-007-2': {
+		author: 'J.K. Rowling',
+		title: {
+			en: 'Harry Potter and the Chamber of Secrets',
+			pl: 'Harry Potter i Komnata Tajemnic ',
+		},
+		translator: {
+			en: null,
+			pl: 'Andrzej Polkowski',
+		},
+	},
 }
 
-books.getAuthor = function(isbn) {
-    // w tym przypadku this === books
-    
-    if(typeof this[isbn] === 'undefined') {
-        // taki ISBN nie istnieje w moim spisie
-        return null;
-    }
-
-    const author = this[isbn]['author'];
-    if(author) {
-        // zwracam informacje o autorze
-        return author;
-    }
-
-    // brak informacji o autorze
-    return false;
-}
+books.getAuthor = function (isbn) {
+	// w tym przypadku this === books
+
+	if (typeof this[isbn] === 'undefined') {
+		// taki ISBN nie istnieje w moim spisie
+		return null
+	}
 
-books.getTitle = function(isbn, lang) {
+	const author = this[isbn]['author']
+	if (author) {
+		// zwracam informacje o autorze
+		return author
+	}
 
+	// brak informacji o autorze
+	return false
 }
 
-books.getTranslator = function(isbn, lang) {
+books.getTitle = function (isbn, lang) {
+	const title = this[isbn]['title'][lang]
```

Comment:
Trzeba sprawdzać to pokolei bo inaczej na tym etapie będzie błąd np. jeśli `this[isbn] === undefined` to `this[isbn]['cokolwiek']` nie istnieje i na tym etapie zatrzyma się działanie programu. My chcemy temu przeciwdziałać dlatego musimy wszystko sprawdzić po kolei tj. najpierw `this[isbn]`, potem `this[isbn]['title'][lang]` (możemy założyć, że `this[isbn]['title']` istnieje)